frontend/src/lib/components/PropertyFilters/utils.ts

```diff
@@ -2,6 +2,16 @@ import { PropertyGroupFilter, AnyPropertyFilter, EventDefinition, PropertyFilter
 import { TaxonomicFilterGroupType } from 'lib/components/TaxonomicFilter/types'
 import { flattenPropertyGroup, isPropertyGroup } from 'lib/utils'
 
+export function sanitizePropertyFilter(propertyFilter: AnyPropertyFilter): AnyPropertyFilter {
+    if (!propertyFilter.type && propertyFilter.value) {
```

Comment:
Why check for `propertyFilter.value` too? Note: "is set"/"is not set" doesn't have to have a `value`.